### ISSUE TITLE
MySQL wire protocol: Nullable fields conversion

### ISSUE THREAD

**Issue by Slach** [ClickHouse/ClickHouse#8299]
When we try SELECT Nullable fields from Clickhouse via MySQL protocol, null value converts as "utf-8" string 'ᴺᵁᴸᴸ' (length=12 bytes 3 bytes per symbol)
**How to reproduce**
```bash
echo "<yandex><mysql_port>33306</mysql_port></yandex>" > /etc/clickhouse-server/config.d/mysql.xml
systemctl restart clickhouse-server

echo "<?$db=mysqli_connect('127.0.0.1:33306','default','');$q=$db->query('SELECT NULL AS test'); foreach($q as $row) { var_dump($row) };" > test.php
php -f test.php
```

* Which ClickHouse server version to use
19.17

* Which interface to use, if matters
MySQL wire protocol

**Expected behavior**
`test` fields should be parsed as php `null` values

**Actual behavior**
![image](https://user-images.githubusercontent.com/105560/71186221-aa909280-229e-11ea-854e-b14d95dcc436.png)
also in internal protocol field type pass as a string
![image](https://user-images.githubusercontent.com/105560/71186467-1541ce00-229f-11ea-885a-34022cd581e6.png)
![image](https://user-images.githubusercontent.com/105560/71188007-fdb81480-22a1-11ea-9bba-765dd7df0e0e.png)

when I run the same query on the native MySQL server
I got NULL value as expected
![image](https://user-images.githubusercontent.com/105560/71186636-6356d180-229f-11ea-8a5f-75ad1e6374e5.png)
on MySQL protocol level, it's represented as
![image](https://user-images.githubusercontent.com/105560/71187560-1bd14500-22a1-11ea-8448-5db430d18e40.png)
![image](https://user-images.githubusercontent.com/105560/71187869-bf225a00-22a1-11ea-9bfe-a2cc76d52284.png)

**Comment by Slach** [comment]
dear @yurriy could you look to this issue?

**Comment by yurriy** [comment]
It is fixed in #7975 
Btw, which program did you use to display protocol packets?

**Comment by filimonov** [comment]
> It is fixed in #7975
> Btw, which program did you use to display protocol packets?

Looks like wireshark
